x-pack/plugin/identity-provider/src/main/java/org/elasticsearch/xpack/idp/rest/RestSamlMetadataAction.java

```diff
@@ -23,11 +23,11 @@
 
 import static org.elasticsearch.rest.RestRequest.Method.GET;
 
-public class RestSamlMetadataRequestAction extends BaseRestHandler {
+public class RestSamlMetadataAction extends BaseRestHandler {
 
     @Override
     public String getName() {
-        return "saml_idp_generate_metadata";
+        return "saml_metadata";
```

Comment:
Tiny nit, our names typically end in `_action`.
TBH I don't know why - it's already obvious it's an action.